### PR TITLE
Pluggable operators

### PR DESCRIPTION
**Pull request by nowsprinting**
### Overview
Implement pluggable Operators. The Operator includes a matcher and operating method.
When you want to perform special operations on a particular Component or GameObject, or when UI elements do not support uGUI, you can define a corresponding operator.

fixed #54 

### Changes
- Add `IOperator` interface
- Implements uGUI operators:`UGUIClickOperator`,`UGUIClickAndHoldOperator`,`UGUITextInputOperator`
- Optimize `MonkeyConfig`
  - Move `TouchAndHoldDelayMillis`, `RandomString`, and `RandomStringParametersStrategy` to operators
  - `IsReachable` strategy method includes ScreenPointStrategy. So remove GetScreenPoint function from MonkeyConfig
- Mod the operation target lottery to be performed by flatly expanding all operators

### TODO
- #116

### CODE REVIEW

**Comment by nowsprinting** [comment]
@Kuniwak san, please review if you have time 🙇

**Comment by Kuniwak** [comment]
This is just a thought. Since IOperator has an IsMatch method (and I think CanOperate would be a more intuitive name, as per the comments), I wondered if it might not be necessary for Component to have something like CanClick.

File: Runtime/Operators/IOperator.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        /// Intended for use in capture and playback features.
+        /// </summary>
+        OperatorType Type { get; }
+
+        /// <summary>
+        /// Returns if can operate target component this Operator.
+        /// </summary>
+        /// <param name="component">Target component</param>
+        /// <returns>True if can operate component this Operator.</returns>
+        bool IsMatch(Component component);
```
[the remark this answers is not in the document]
Comment (nowsprinting) [reply Runtime/Operators/IOperator.cs]: Thanks! Rename it, at 3b9600b

**Comment by nowsprinting** [comment]
@Kuniwak san, Thank you for your review!
I obsolesce wrapper methods (e.g., CanClick) at f100642

**Pull request merged**